src/app/containers/NewStoryPromo/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
```

Comment:
Should we move `NewStoryPromo` and `NewStoryPromoList` to `components/OptimoPromos` because:
- We are no longer using the container component pattern going forward
- It keeps all the experimental code together to be reorganised later in line with the component standards

Reply:
Yes, also easy to have the code all in one place 👍

Reply:
Done